Lecture_1/task_1/src/main/java/main.java

```diff
@@ -0,0 +1,52 @@
+import java.util.Scanner;
+
+class task_1 {
```

Comment:
read https://www.javatpoint.com/java-naming-conventions
class name should be From Upper_Case and without spec  symbols